lib/ui/platform_dispatcher.dart

```diff
@@ -1300,10 +1305,12 @@ class ViewConfiguration {
     this.padding = WindowPadding.zero,
     this.gestureSettings = const GestureSettings(),
     this.displayFeatures = const <DisplayFeature>[],
-  });
+  }) : assert(!(window != null && view != null)),
+    _view = view ?? window;
 
   /// Copy this configuration with some fields replaced.
   ViewConfiguration copyWith({
+    FlutterView? view,
     FlutterView? window,
```

Comment:
copy with should do something with the window property instead of silently ignoring it.